tutorials/00_introduction.ipynb

```diff
@@ -96,7 +96,7 @@
    "id": "0a5ca5f5",
    "metadata": {},
    "source": [
-    "To print the metadata of the program `sample-program`:"
+    "You can pass `detailed = True` parameter to `pprint_programs()` to view all the metadata for the available programs. The program metadata once fetched, is cached for performance reasons, so you can pass `refresh = True` parameter to `pprint_programs()` or `programs()` methods in order to get the latest programs from the server. By default first 20 programs are displayed. You can use the `limit` and `skip` parameters in `pprint_programs()` and `programs()` to page through the programs. To print the metadata of the program `sample-program`:"
```

Comment:
So `pprint_programs()` doesn't actually print metadata for _all_ available programs as claimed in this paragraph and earlier? Just the first 20?